i18n/translation-en.json

```diff
@@ -197,6 +197,15 @@
       "text": "Text",
       "colors": "Colors",
       "resetAllSettings": "Reset All Settings"
+    },
+    "thumbnailPanel": {
+      "deleteWarningTitle": "Delete Page",
+      "deleteWarningMessage": "Are you sure you want to delete the selected page(s). This can't be undone",
```

Comment:
So something like
```
  "warning": {
    "deletePage" :{
      "deleteTitle": "Delete Page",
      "deleteMessage": "Are you sure you want to delete the selected page(s). This can't be undone",
      "deleteLastPageMessage": "This would result in no pages remaining. Please leave at least one page",
    },
    "redaction":{
      "applyTile": "Apply Redaction",
      "applyMessage": "This action will permanently remove all items selected for redaction. It cannot be undone.",
    }
  },
```
And we call it with `warning.deletePage.deleteMessage`. Or were you thinking it would be `option.warningModel.deleteMessage` or `option.warningModal.redactionApplyMessage`

Reply:
Yeah I think either of them is fine. Do you think it make sense?

Reply:
I ended up moving the message to the top level so it's now `warning.deletePage.deleteMessage`